src/sign-in-blazorwasm/wwwroot/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+    <title>blazorwasm</title>
+    <base href="/" />
+    <link href="css/bootstrap/bootstrap.min.css" rel="stylesheet" />
+    <link href="css/app.css" rel="stylesheet" />
+    <link href="blazorwasm.styles.css" rel="stylesheet" />

```

Comment:
I don't know much about how blazor WASM works, but curious where the `blazorwasm.styles.css` comes from (I don't see on the filesystem) and how that's related to `css/app.css` (which I see on file system)

Reply:
:eyes:

<img width="323" alt="image" src="https://user-images.githubusercontent.com/158487/146590706-898cb5ac-4089-4b6d-adcf-9dd958c41027.png">

Reply:
Good to know!